tests/test_ilastik/test_applets/layerViewer/testLayerViewerGui.py

```diff
@@ -0,0 +1,75 @@
+import numpy
+import vigra
+
+from ilastik.applets.layerViewer.layerViewerGui import LayerViewerGui
+from lazyflow.graph import Graph
+from lazyflow.operator import Operator
+from lazyflow.slot import OutputSlot, InputSlot
+
+
+class OpPassthrough(Operator):
+    """Op with a slot for LayerViewerGui to put in self.observedSlots"""
+
+    Image = InputSlot()
+    Output = OutputSlot()
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.Output.connect(self.Image)
+
+    def propagateDirty(self, slot, subindex, roi):
+        pass
+
+
+def make_simple_op():
+    op = OpPassthrough(parent=Operator(graph=Graph()))
+    op.Image.setValue(numpy.array([[1, 1], [1, 1]]))
+    op.Output.meta.axistags = vigra.defaultAxistags("xy")
+    return op
+
+
+def test_layer_update_respects_2d_default():
```

Comment:
for completeness, this test could be parametrized to also test the `prefer_2d=False` case.